syft/pkg/cataloger/swift/test-fixtures/empty-packages.resolved

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
The issue at https://github.com/anchore/syft/issues/2225 mentions an "empty file" (make with `touch Packages.resolved`), but this is an empty JSON document. Is it worth adding a test that makes sure syft doesn't panic when parsing an empty file?

Reply:
done!